runtime/tools/python/test/test_check.py

```diff
@@ -27,151 +28,214 @@
 def test_clean_artifacts():
     API.initialize_apis()
     custom_args = {}
+    custom_args[
+        "--result-file"
+    ] = f"ttrt-results/{inspect.currentframe().f_code.co_name}_check.json"
     custom_args["binary"] = BINARY_FILE_PATH
     custom_args["--clean-artifacts"] = True
     check_instance = API.Check(args=custom_args)
     check_instance()
 
     assert (
-        check_results("check_results.json") == 0
-    ), f"one of more tests failed in={test_clean_artifacts.__name__}"
+        check_results(
+            f"ttrt-results/{inspect.currentframe().f_code.co_name}_check.json"
+        )
+        == 0
+    ), f"one of more tests failed in={tinspect.currentframe().f_code.co_name}"
```

Comment:
This check seems to repeat in all tests, maybe you can avoid copy paste by making common assert, or maybe 
[pytest_runtest_teardown / pytest_runtest_teardown](https://docs.pytest.org/en/latest/reference/reference.html#test-running-runtest-hooks) hooks can be useful here to keep things [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

Reply:
fixed